frontend/packages/knative-plugin/src/components/overview/__tests__/ConfigurationsOverviewList.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { sampleKnativeConfigurations } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
```

Comment:
This import shouldn't happen. We shouldn't be pulling knative test data from `dev-console` into `knative-plugin`. 

But since there are a ton that exist prior to you, I'll let it go. We should definitely put test data with the proper package. There are about a dozen imports from `knative` tests to get data from `dev-console` and only a couple using relative pathing to get data for tests in `dev-console`. Even though this is topology related data, I feel it belongs in `knative-plugin`.

cc @christianvogt

Reply:
@andrewballantyne 
we can have a tech debt story, to move such data to a shared space, probably `@console/shared`

Reply:
Yup, a tech debt ticket to clean up all imports is a good idea. I am not sure they should move to `@console/shared` - they are knative related, so I suspect the best spot for them is the knative package. `dev-console` can pull from `knative` the inverse is not true.